frontend/packages/shared/src/components/LeftNavigationBar/LeftNavigationBar.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ReactNode, useState } from 'react';
+import classes from './LeftNavigationBar.module.css';
+import { LeftNavigationTab } from 'app-shared/types/LeftNavigationTab';
+import { GoBackButton } from './GoBackButton';
+import { Tab } from './Tab';
+
+export type LeftNavigationBarProps = {
+  /**
+   * List of navigation tabs
+   */
+  tabs: LeftNavigationTab[];
+  /**
+   * The upper tab
+   */
+  upperTab?: 'backButton' | 'searchField' | undefined;
```

Comment:
Er dette om den skal vise en tilbake knapp eller et søkefelt? :)

Reply:
Ja, tanken var at man kan legge til forskjellige ting som skal vises som den øverste taben, med default til undefined/null slik at man kan bruke bare tabs uten å ha en special case for den øverste. 
Grunnen til at jeg la til ```backButton``` er at resourceadm bruker en back button i navigasjonsbaren. 
Grunnen til at jeg la til ```searchField``` er at UX designet til apps viser at det er et søkefelt der. 

Kanskje navnet burde endres til noe mer forklarende?

Reply:
Har vi tenkt noe på hvordan dette søkefeltet skal fungere? Har vi noen tanker om hvordan vi skal implementere logikken for søk? Eller burde vi droppe alt som har med søkefelt i denne PRen og heller lage en egen issue som inneholder avklaringer på hvordan søkefeltet eventuelt skal fungere?

Hva skal skje når man søker i søkefeltet? Hvordan ser det ut dersom det ikke finnes noen treff på søkeord? Jeg foreslår at vi dropper å skrive kode som implementerer noe med søkefelt å gjøre, men heller skrive kode som lar seg utvide i fremtiden. 

Kanskje UX også får andre tanker når vi begynner å utfordre på de forskjellige scenarioene som på tenkes på. :)

Reply:
Kan godt fjerne det ja, så kan vi legge det til når vi vet hvordan det vil se ut endelig :)